.envrc

```diff
@@ -1,41 +1,27 @@
-export SERVER_ROOT=$(pwd)
+### General env vars ###
+export WORKSPACE_ROOT=$(pwd)
 export RUST_LOG_FORMAT=devel
 export RUST_BACKTRACE=1
 export RUST_LOG=query_engine_tests=trace,query_engine=debug,query_core=trace,query_connector=debug,sql_query_connector=debug,prisma_models=debug,engineer=info,sql_introspection_connector=debug,tiberius=trace,quaint=debug
-export SKIP_CONNECTORS="vitess_5_7,vitess_8_0"
-
-# Controls Scala test kit verbosity. Levels are trace, debug, info, error, warning
-export LOG_LEVEL=trace
-
-# Controls Scala test kit test complexity ("simple", "complex")
-export TEST_MODE=simple
-
-# Controls scala test kit binary build mode ("debug", "release"): Defaults to "release" if not set.
-export TEST_BINARY_BUILD_MODE=debug
 
-# Local dev env
+### Various local dev env vars ###
 export PRISMA_CONFIG_PATH=$(pwd)/prisma.yml
 export PRISMA_DML_PATH=$(pwd)/datamodel_v2.prisma
-
 export PRISMA2_BINARY_PATH="/usr/local/lib/node_modules/prisma2/"
```

Comment:
I don't use this file usually so I don't have any strong opinion, but I'm curious, what is this variable for?

Reply:
This is something that the makefile uses 🤷